src/Lib/QueryProcessor.php

```diff
@@ -27,8 +29,8 @@ class QueryProcessor {
 	// We set this on initialization (init.php) so we are sure we have it in class
 	protected static ContainerInterface $container;
 
-	/** @var bool */
-	protected static bool $inited = false;
+	/** @var ?array<mixed,mixed> $searchdSettings */
```

Comment:
Lets make proper annotation with settings available in show settings response to make life easier when we will use something else in future